chromium_src/third_party/blink/renderer/core/execution_context/execution_context.cc

```diff
@@ -147,7 +141,8 @@ AudioFarblingCallback BraveSessionCache::GetAudioFarblingCallback(
       }
       case BraveFarblingLevel::MAXIMUM: {
         uint64_t seed = *reinterpret_cast<uint64_t*>(domain_key_);
-        return base::BindRepeating(&PseudoRandomSequence, seed);
+        auto prng = std::mt19937_64(seed);
+        return base::BindRepeating(&PseudoRandomSequence, base::OwnedRef(prng), seed);
```

Comment:
@goodov so this compiles and passes tests, but `float PseudoRandomSequence(std::mt19937_64& prng, ...)` above fails `npm run lint` because of the non-const reference. Despite that [being a example](https://source.chromium.org/chromium/chromium/src/+/main:base/bind.h;l=200-211) in the docs.

What would you recommend here? Maybe
```c++
float PseudoRandomSequence(std::mt19937_64* prng, ...)
...
auto* prng = new std::mt19937_64(seed);
return base::BindRepeating(&PseudoRandomSequence, base::Owned(prng), seed);
// or maybe...
return base::BindRepeating(&PseudoRandomSequence, base::Unretained(prng), seed);
```
Seems worse, but lint doesn't complain. I could wrap it in a RefCounted an pass it as `OwnedRef` but that's the same as `std::shared_ptr`?

Reply:
`base::Owned` should do the trick. `Unretained` will leak the pointer.
and seems like you can just `seed()` the generator instead of reconstructing it when.

Reply:
Ok, thanks! Good point on the `seed()` method.